cartridge/remote-control.lua

```diff
@@ -414,14 +432,18 @@ end
 
 --- Explicitly drop all established connections.
 --
+-- Close all the sockets except the one that triggered the function.
+-- The last socket will be closed when all requests are processed.
+--
 -- @function drop_connections
 -- @local
 local function drop_connections()
     local handlers = table.copy(vars.handlers)
     table.clear(vars.handlers)
```

Comment:
You're right. It can be rewritten without affecting the behaviour.

I usually prefer to use `table.clear` because it seems to be safer. It's nearly impossible to check (autotest) that no other code makes a closure on the table. And if it did, it could result in a very rare race which is really hard to catch. It could be something simple like

```lua
for s, handler in pairs(vars.handlers) do
    -- implicit fiber.yield()
end
```

But this is a general consideration, and it doesn't apply to this particular case. I can change it if you still want to.

Reply:
Well, I don't insist. Let's stay as is.